### ISSUE TITLE
Issues when training with the pretrained autoencoder (cfg_kitti_fm.py)

### ISSUE THREAD

**Issue by ruili3** [TRI-ML/packnet-sfm#120]
I train the network using the setting of pretrained fixed autoencoder (cfg_kitti_fm.py), and during the training procedure, I found in the log that the min perceptual loss (and smooth loss) keep to be zeros across all scales, while the reconstruction loss seems to be right. The log of one line is shown below:

> 2021-02-15 11:14:17,379 - INFO - Epoch [1][450/2489]	lr: 0.00009, eta: 5 days, 0:02:05, time: 2.996, data_time: 0.027, memory: 5717, ('min_reconstruct_loss', 0): 0.0328, ('min_perceptional_loss', 0): 0.0000, ('smooth_loss', 0): 0.0000, ('min_reconstruct_loss', 1): 0.0328, ('min_perceptional_loss', 1): 0.0000, ('smooth_loss', 1): 0.0000, ('min_reconstruct_loss', 2): 0.0328, ('min_perceptional_loss', 2): 0.0000, ('smooth_loss', 2): 0.0000, ('min_reconstruct_loss', 3): 0.0327, ('min_perceptional_loss', 3): 0.0000, ('smooth_loss', 3): 0.0000, loss: 0.1312

I wonder is there anything wrong in my setting to run the code? Or it is natual (the loss is super small) during the right training procedure?

Thanks a lot!
